summoner-cli/src/Summoner/Config.hs

```diff
@@ -155,14 +155,14 @@ configT = Config
     lastT :: (Key -> TomlCodec a) -> Key -> TomlCodec (Last a)
     lastT codec = Toml.dimap getLast Last . Toml.dioptional . codec
 
-    _GhcVer :: BiMap GhcVer Toml.AnyValue
-    _GhcVer = Toml._TextBy showGhcVer parseGhcVer
+    _GhcVer :: TomlBiMap GhcVer Toml.AnyValue
+    _GhcVer = Toml._TextBy showGhcVer (maybeToRight "Wrong GHC version" . parseGhcVer)
```

Comment:
Sorry for inconvenience... I guess we need more helper functions in `tomland` to write such things easier...